src/components/catalog/PipelineCatalog.tsx

```diff
@@ -67,6 +72,9 @@ const PipelineCatalog = () => {
           }
         })
       }
+      setTimeout(() => {
+        setFetchinData(false)
+      }, 2000)
```

Comment:
How come the loading screen is hard-coded to be turned off after 2 seconds?

Reply:
I guess I was just trying to give the user a bit of a wait cause sometimes the data fetches rapidly and it looks like a bit of a snap. I would make the necessary Changes